cmsimple/tplfuncs.php

```diff
@@ -249,14 +249,28 @@ function searchbox()
 }
 
 
+/**
+ * Returns a link to the homepage.
+ *
+ * @param string $img Optional image file name, searched in template folder.
+ *
+ * @return string HTML
+ */
+function homelink($img = '') 
+{
+    //TODO why not use ml() ?
+    return a(0) . img_or_text($img, 'Home') . '</a>';
```

Comment:
Instead of the hard-coded zero, it seems to be more appropriate to use `xh_publisher->getFirstPublishedPage()` here.

Reply:
I don't understand how to use xh_publisher